src/Simplex/Chat/Messages/CIContent.hs

```diff
@@ -328,6 +329,7 @@ rcvDirectEventToText :: RcvDirectEvent -> Text
 rcvDirectEventToText = \case
   RDEContactDeleted -> "contact deleted"
   RDEProfileUpdated {} -> "updated profile"
+  RDEGroupInvLinkReceived -> "requested connection from group"
```

Comment:
here we could include current group profile, as we do for member profiles in many places